pillarbox-player/src/main/java/ch/srgssr/pillarbox/player/asset/BlockedInterval.kt

```diff
@@ -0,0 +1,55 @@
+/*
+ * Copyright (c) SRG SSR. All rights reserved.
+ * License information is available from the LICENSE file.
+ */
+package ch.srgssr.pillarbox.player.asset
+
+import android.os.Parcel
+import android.os.Parcelable
+
+/**
+ * Blocked interval
+ *
+ * @property id the id of the chapter.
+ * @property start the start position.
+ * @property end the end position.
+ * @property reason the block reason.
+ * @constructor Create empty Blocked interval
+ */
+data class BlockedInterval(
+    override val id: String,
+    override val start: Long,
+    override val end: Long,
+    val reason: String
```

Comment:
Should we use an enum here?

Reply:
We are in the Player module, so we don't have any clue about business blocking reason. We could instead replace it with a generic that have to be parcelable or serializable.